test/access-token.test.js

```diff
@@ -462,6 +473,11 @@ describe('loopback.token(options)', function() {
 describe('AccessToken', function() {
   beforeEach(createTestingToken);
 
+  it('should have getAccessTokenId method', function() {
+    assert(typeof Token.getAccessTokenId === 'function');
+    assert(typeof Token.findForRequest === 'function');
+  });
```

Comment:
- there should be one assertion per test (test on `.findForRequest()` is unrelated)
- new tests should use expect, not assert
- new test descriptions do not mention 'should': `it has getAccessTokenId method`
- test on `.getAccessTokenId()` should actually verify that the method can be customized by the user (main purpose on the PR)
- while at it, test on `. findForRequest()` should do likewise (as it's public api)